lang/fr.json

```diff
@@ -94,6 +98,7 @@
   "submission.error": "Une erreur est survenue. Veuillez renvoyer votre réponse.",
   "about": "À Propos",
   "home": "Accueil",
+  "sourcecode": "Code source",
   "about.collection.title": "Qu'est-ce que ce site collecte ?",
   "about.collection.content": "Ne sont collectées que les données que vous entrez dans le formulaire. Rien d'autre.",
   "about.utility.title": "Pourquoi est-ce utile ?",
```

Comment:
Small error in ** Mentions** : 
`a titre gracieux`  -> `à titre gracieux`

Reply:
Fixed in c9f8302